### ISSUE TITLE
Miniimagenet Results?

### ISSUE THREAD

**Issue by cinjon** [tristandeleu/pytorch-meta#44]
Have you successfully used this library to train Miniimagenet with Prototypical Nets? The highest test accuracy that I have gotten was .52, and that was changing the transforms to additionally Normalize the image tensors and add in a step size scheduler that advances every 20 epochs.

**Comment by cinjon** [comment]
For reference, I tried the same settings using this library https://github.com/cyvius96/prototypical-network-pytorch and got the expected results. Having a lot of trouble getting miniimagenet to work with this library.

**Comment by carmete** [comment]
The training parameters for 5-shot, 5-way are different to what is reported in Snell et al. (2017):

For meta-training:
- --num_ways 20
- --batch_size 1 (higher batch size might improve results)
- --num_batches 10000 (Snell does early stopping with patience 200)
- 5 support_shots / 15 query_shots
- learning rate of 0.001 decayed by factor 0.5 every 2000 episodes (/batches)

For meta-testing:
- --num_ways 5
- rest as for training

Does this work? It should give something around 66%, which is still a bit lower than the reported results.

**Comment by brando90** [comment]
has anyone tried it with MAML?
